guides/v2.2/cloud/howtos/debug.md

```diff
@@ -94,79 +84,94 @@ When deployed to Starter environments and Pro Integration environments, Xdebug i
+
+1. In your PhpStorm project, open the settings panel.
+
+   -  _Mac OS X_—Select **File** > **Preferences**.
+   -  _Windows/Linux_—Select **File** > **Settings**.
+
+1. In the _Settings_ panel, expand and locate the **Languages & Frameworks** > **PHP** > **Servers** section.
 
-   *  On Mac, select **File** > **Preferences**.
-   *  On Windows/Linux, select **File** > **Settings**.
+1. Click the **+** to add a server configuration. The project name is in grey at the top.
 
-1. Expand and locate **Languages & Frameworks** > **PHP** > **Servers** section in settings.
-1. Add a server configuration. Click the + to add a server. Notice at the top, it will show the project name in grey, just for reference. This will create a "server" configuration. This will be used to listen to port 9000 locally, which will be [forwarded](#port).
-1. Configure settings for the new server:
+1. Configure the following settings for the new server configuration:
 
-   *  **Name**: enter the same as the hostname. This value is used in and must match the value for `PHP_IDE_CONFIG` variable in [Debug CLI commands](#debugcli).
-   *  **Host**: enter `localhost`
-   *  **Port**: enter 80
-   *  **Debugger**: set to Xdebug in the drop-down
+   -  **Name**—enter the same as the hostname. This value is used in and must match the value for `PHP_IDE_CONFIG` variable in [Debug CLI commands](#debugcli).
+   -  **Host**—Enter `localhost`.
+   -  **Port**—Enter `80`.
+   -  **Debugger**—Select `Xdebug`.
+
+1. Select **Use path mappings**. In the _File/Directory_ pane, the root of the project for the `serverName` displays.
 
-1. Select the **Use path mappings** option. In the files/directories, the root of the project displays that you opened for the added server.
 1. In the **Absolute path on the server** column, click ![Edit]({{ site.baseurl }}/common/images/install_docker_php-storm-edit.png){:width="15px"} (**Edit**) and add a setting based on the environment:
 
-   *  For all Starter environments and Pro Integration environments, the remote path is `/app`.
-   *  For Pro Staging and Production environments:
+   -  For all Starter environments and Pro Integration environments, the remote path is `/app`.
+   -  For Pro Staging and Production environments:
+
+      -  Production: `/app/<project_code>/`
+      -  Staging:  `/app/<project_code>_stg/`
 
-      *  Production: `/app/<project_code>/`
-      *  Staging:  `/app/<project_code>_stg/`
+1. Change the Xdebug port to 9001 in the **Languages & Frameworks** > **PHP** > **Debug** > **Xdebug** > **Debug Port** panel.
+
+1. Click **Apply**.
 
 ### Set up port forwarding {#port}
 
-You need to set up port forwarding. This is necessary to map the XDEBUG connection from the server to your local system to PHPSTORM and xdebug locally can track along with it
+You need to set up port forwarding. This is necessary to map the XDEBUG connection from the server to your local system to PHPSTORM and Xdebug locally can track along with it
```

Comment:
Last part of sentence "and Xdebug locally can track along with it"  -- not sure what this means. Maybe missing comma, or maybe "and" is "so that"?  Also, missing period at end of 2nd sentence.

Reply:
Yeah, yuck.